development -> main

Haroldo-Mayworm opened the pull request.

Haroldo-Mayworm committed:
refactor(design): Redesign of all styles

Haroldo-Mayworm committed:
refactor(design): Details in titles

bribeiro commented on css/global/cores.css:
```diff
@@ -1,9 +1,12 @@
-  --cor-bege: #F5F0F1;
-  --cor-azulClaro: #28A4DD;
-  --cor-fundo: #304977;
+  --cor-laranja: #f0340bc7;
```
talvez você não precise do "cor", o nome do arquivo descreve o que são essas variaveis. Você pode mudar pra 
`--branco`, `--preto`, etc. Além disso, talvez você queira variaveis para as cores mais especificas como o azul claro. no código se você entrar `color: white` vai funcionar.

bribeiro commented on css/global/tipografia.css:
```diff
@@ -0,0 +1,4 @@
+:root {
+  --font-principal: Comfortaa, sans-serif;
```
Eu acredito que essa primeira fonte não seja de sistema, onde você esta carregando ela?

bribeiro replied:
achei. Talvez você prefira carregar ela nesse modo: https://stackoverflow.com/questions/14676613/how-to-import-google-web-font-in-css-file

bribeiro commented on index.html:
```diff
@@ -31,29 +31,28 @@
       <ul class="menu-lista">
         <li class="menu-lista_item">
-          <a class="menu-lista_link" href="#"><img src="img/header/nav-sobre.png" alt="">Sobre</a>
+          <a class="menu-lista_link" href="#sobre"><img src="img/header/nav-sobre.svg" alt="">Sobre</a>
```
sempre tenha o alt preenchido. Isso é o que os leitores de tela usam para dizer ao usuário do que se trata aquela imagem

bribeiro replied:
outra coisa que você pode explorar, se a imagem não é parte do conteúdo, é colocar a imagem no css.

bribeiro commented on index.html:
```diff
@@ -184,7 +182,7 @@ <h2>Contato</h2>
 
   <footer>
     <div class="footer">
```
por que você precisa dessa div?

bribeiro reviewed (commented):
Um outro comentário é com relação a esse bloco: https://github.com/Haroldo-Mayworm/Haroldo_Mayworm/blob/development/index.html#L24-L26 Você deveria estar usando h1, já que é o titulo da pagina.

https://github.com/Haroldo-Mayworm/Haroldo_Mayworm/blob/development/index.html#L57 deveria ser um h2 e https://github.com/Haroldo-Mayworm/Haroldo_Mayworm/blob/development/index.html#L58 h3

Haroldo-Mayworm committed:
style(css): Aperfeiçoa a estilização e nomeclaturas

Ajustei os ícones do menu, ajustei a importação da fonte e alterei alguns nomes de variáveis

Haroldo-Mayworm committed:
feat(Style): Added tablet responsive styles

Implemented responsive styles to optimize the site's appearance and usability on tablets. This includes adjustments to: layout, font sizes and other elements.

Haroldo-Mayworm committed:
feat(style): Added responsive styles for computers

Responsive desktop styles implemented. This includes adjustments to: layout, font sizes and other elements.
